src/locale/locales/pt-BR/messages.po

```diff
@@ -7056,7 +7056,7 @@ msgstr "Os Termos de Serviço foram movidos para"
 
 #: src/components/intents/VerifyEmailIntentDialog.tsx:85
 msgid "The verification code you have provided is invalid. Please make sure that you have used the correct verification link or request a new one."
-msgstr ""
+msgstr "O código de verificação que você forneceu é inválido. Certifique-se de que você usou o link de verificação correto ou solicite um novo."
```

Comment:
```suggestion
msgstr "O código de verificação que você forneceu é inválido. Certifique-se de que você usou o link de verificação correto ou solicite novamente."
```

Reply:
For me, both are correct. The second one is more common but the first one is correct. It depends a lot on the context.